poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
@sy-cui will your fix for Sopht works with poetry 1.6.1 ? Should we bump poetry version? They should be irrelevant I think, can you confirm?

Reply:
I won't fix it for now but this should be fine. I tested it with a local 'pip install', with lock file in 1.6.1 but poetry in 1.5.1. Pyelastica is installed without problem

Reply:
Okay thanks